R-package/R/xgb.Booster.R

```diff
@@ -1206,6 +1206,14 @@ xgb.copy.Booster <- function(model) {
   return(.Call(XGDuplicate_R, model))
 }
 
+xgb.reset.Booster <- function(model) {
```

Comment:
Does this function also clean up memory produced during predictions? If so, perhaps it could also get exported.

Reply:
Let's keep it internal for now. I don't want others to abuse it as it has a huge overhead.